module/zfs/dsl_scan.c

```diff
@@ -3866,6 +4181,54 @@ dsl_scan_scrub_cb(dsl_pool_t *dp,
 	return (0);
 }
 
+static void
+read_by_block_level(dsl_scan_t *scn, zbookmark_phys_t zb)
```

Comment:
Why define this below where it's used and have a separate declaration when you could instead just define it above where it's used?